sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListFileVersionExpirationReportJobProgress.md

```diff
@@ -0,0 +1,99 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spolistfileversionexpirationreportjobprogress
+applicable: SharePoint Online
+title: Get-SPOListFileVersionExpirationReportJobProgress
+schema: 2.0.0
+author: msjennywu
+ms.author: jennywu
+ms.reviewer:
+manager: seanmc
+---
+
+# Get-SPOListFileVersionExpirationReportJobProgress
+
+## SYNOPSIS
+
+Gets the status for a file version usage report generation job for a document libray.
+
+## SYNTAX
+
+```powershell
+Get-SPOListFileVersionExpirationReportJobProgress [-Site] <SpoSitePipeBind>
+ [List] <SpoListPipeBind>
+ [-ReportUrl <String>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Gets the status for a file version usage report generation job for a document libray.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Get-SPOListFileVersionExpirationReportJobProgress -Site https://contoso.sharepoint.com/sites/site1 -List "Documents" -ReportUrl "https://contoso.sharepoint.com/sites/sites1/reports/MyReports/VersionReport.csv"
```

Comment:
>ReportUrl [](http://example.com/codeflow?start=118&length=9)

is there any relationship between this ReportUrl and the List param?

Reply:
The path of -ReportUrl can be in any document library of the same site

Reply:
great! that's why I was asking :D